libraries/engage/core/config/theme-config.schema.json

```diff
@@ -0,0 +1,434 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "color": {
+      "type": "string",
+      "pattern": "^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$"
+    },
+    "shadow": {
+      "type": "string",
+      "minLength": 1
+    },
+    "icon": {
+      "type": "string",
+      "minLength": 1,
+      "pattern": "^(<([^>]+)>)$"
```

Comment:
This pattern doesn't match the most of our icons. It only matches to strings which contain a single XML tag, but a lot of our icon strings contain multiple tags .

Reply:
Fixed.